test_sets/ExternalUserConsentStatus_Informing_HMI.txt

```diff
@@ -0,0 +1,9 @@
+./test_scripts/Polices/External_UCS_Informing_HMI/001_ATF_P_GetListofPermissions_with_appID_all_ecs_in_bound_EntitySatus_ON.lua Check_that_SDL_returns_all_params_in_bound_from_OnAppPermissionConsent_notification_Entity_Status_ON

```

Comment:
@istoimenova Tests covering [HMI RPC validation]: SDL behavior: HMI sends request (response, notification) with fake parameters that SDL should use internally are added